### PR TITLE
First version of prepare role

### PR DESCRIPTION
**Pull request by greg-hellings**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+
+After invoking this role, the adm host will be ready for invoking the openshift-ansible
+playbooks against the other hosts in the cluster. An invocation such as the following
+should do the trick
+
+```bash
+cd /usr/share/ansible/openshift-ansible/playbooks
+# export ANSIBLE_CONFIG=/etc/ansible/ansible.cfg
+ansible-playbook prerequisites.yml
+ansible-playbook deploy_cluster
```
Comment (andrew103) [inline README.md]: I believe "deploy_cluster" is a YAML file as well.

Comment (greg-hellings) [reply README.md]: True. Fixed!

**Review by andrew103 — APPROVED** [review]

**Review by andrew103 — COMMENTED** [review]
Should the docker scenario stuff be removed here? Noticed it was removed in ocp_configure_users but not here.

**Comment by greg-hellings** [comment]
RE: docker scenario. There's still the potential that I can find a repository that includes the "openstack-installer" repo and use that scenario to build out full testing of this. I just haven't bothered look up the RDO repo URL yet.

File: README.md [hunk 2]
```diff
@@ -30,7 +67,17 @@ Currently the following variables are supported:
...
+to include the necessary repositories as described in the documentation. Additionally the local
+Ansible inventory and group\_vars folders need to already exist on the local system so that they
+can be uploaded to the remote target. This role will install the "openshift-ansible" package, so
+whichever repositories provide the particular version of that package to correspond with the version
+of OCP you are trying to deploy.
+
+Additionally, you need to have a local copy of the SSH private key file that will be used to
+log in to the remote hosts, if these are not explicitly defined elsewhere in group\_vars and
+the inventory file. Obviously, if other methods of authentication are in use, then those are
+outside of the scope of this playbook.
```
Comment (tehsmyers) [inline README.md]: `s/playbook/role`? Other auth methods are presumably outside the scope of this role, but probably still inside the scope of a playbook using this role.

**Review by tehsmyers — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (greg-hellings) [reply README.md]: Good call

**Pull request merged**